### PR TITLE
Fix bugs in export

### LINKED ISSUE #56: Export button doesn't work
On my main test account, clicking on the "Export" button in the sidebar leads to a blank grey page and a console error.

**Steps to reproduce:**

1. Go to contacts
2. Click on the export button in the sidebar
3. Notice the console error and failed event

Do contact me for test account info if not reproducible on your account.

**Example:**

### PR DESCRIPTION
**Pull request by econdepe**
Apart from the bug described in #56, keys were not loaded when the export started, leading to errors in the decryption of some contacts and therefore their export. Both things are fixed. I also fixed some React warnings regarding closing the modal.

Closes #56

### CODE REVIEW

File: src/app/components/ExportModal.js [hunk 1]
```diff
@@ -76,25 +78,25 @@ const ExportModal = ({ contactGroupID: LabelID, onClose, ...rest }) => {
...
                 }
                 try {
-                    const { properties: contactDecrypted = [], errors = [] } = await prepareContact(
+                    const { properties: contactDecrypted = [], errors = [] } = await decrypt(
                         { Cards },
                         { publicKeys, privateKeys }
                     );
 
                     if (errors.length) {
+                        console.log('any error decrypting?', errors);
```
Comment (EpokK) [inline src/app/components/ExportModal.js]: Remove it please.

File: src/app/components/ExportModal.js [hunk 2]
```diff
@@ -16,10 +16,11 @@ import { queryContactExport } from 'proton-shared/lib/api/contacts';
 import downloadFile from 'proton-shared/lib/helpers/downloadFile';
 import { wait } from 'proton-shared/lib/helpers/promise';
 
-import { bothUserKeys, prepareContact } from '../helpers/decrypt';
+import { bothUserKeys, prepareContact as decrypt } from '../helpers/decrypt';
```
Comment (EpokK) [inline src/app/components/ExportModal.js]: I don't like to rename function, it's a pain to maintain if we decide to change something related to this function. If you feel the name should be change, change it everywhere but don't do that please.

Comment (econdepe) [reply src/app/components/ExportModal.js]: Ok. It's not that necessary to change it here. There is one `prepareContact` in `decrypt.js` and another in `encrypt.js`, so in other places that I had to use both I had to do the renaming. I thought it was more readable like that, but indeed the renaming is not necessary here

Comment (EpokK) [reply src/app/components/ExportModal.js]: Thanks.

File: src/app/components/ExportModal.js (hunk 1 above)
Comment (econdepe) [reply src/app/components/ExportModal.js]: this one escaped lint somehow...

**Pull request merged**